dev-python/PythonQt/PythonQt-3.1.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit qmake-utils python-r1
+
+DESCRIPTION="A dynamic Python binding for the Qt framework"
+HOMEPAGE="http://pythonqt.sourceforge.net/"
+SRC_URI="mirror://sourceforge/pythonqt/pythonqt/${PN}-${PV}/${PN}${PV}.zip"
```

Comment:
Also, you can define:

    MY_P=${PN}${PV}

and use it in SRC_URI and S. That's a common practice when upstream uses package name that's different from ${PN}-${PV}.